src/containers/ProfilePage/SectionYoutubeVideoMaybe.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import css from './ProfilePage.module.css';
+import classNames from 'classnames';
+
+import { lazyLoadWithDimensions } from '../../util/uiHelpers.js';
+
+import { AspectRatioWrapper } from '../../components/index.js';
+import { Heading } from '../../components';
+
+const SectionYoutubeVideoMaybe = props => {
+  const { videoUrl, heading } = props;
+
+  const extractYouTubeID = url => {
+    const regExp = /(?:youtube\.com\/(?:[^\/\n\s]+\/\S+\/|(?:v|e(?:mbed)?|shorts)\/|\S*?[?&]v=)|youtu\.be\/)([a-zA-Z0-9_-]{11})/i;
+    const match = url.match(regExp);
+
+    return match ? match[1] : null;
+  };
+
+  const extractedVideoID = extractYouTubeID(videoUrl);
+
+  const RADIX = 10;
+  const BLACK_BG = '#000000';
```

Comment:
Code convention: constants should be defined at the beginning of the file (after imports).
https://github.com/sharetribe/web-template/blob/main/src/components/Avatar/Avatar.js#L19-L21